src/main/kotlin/no/nav/pensjon/elsam/minibuss/tjenestepensjon/TjenestepensjonService.kt

```diff
@@ -55,6 +57,18 @@ class TjenestepensjonService(
         return tjenestepensjon.forhold.isNotEmpty()
     }
 
+    fun slettTjenestepensjonsforhold(fnr: String, tpNr: String) {
+        try {
+            tpRestClient.delete()
+                .uri("/api/samhandler/tjenestepensjon/forhold/$tpNr")
+                .header("fnr", fnr)
+                .retrieve()
+                .body<String>()
+        } catch (e: NotFound) {
+            throw SlettTPForholdFaultTjenestepensjonForholdIkkeFunnetMsg(e.message, e.rootCause)
```

Comment:
Dette returnerer f eks;
`<faultstring>404 Not Found: "{"timestamp":"2024-10-03T10:23:38.902+00:00","status":404,"error":"Not Found","message":"Forhold ikke funnet.","path":"/api/samhandler/tjenestepensjon/forhold/2956"}"</faultstring>`
i stedet for: 
`<faultstring>{"timestamp":"2024-10-03T10:02:50.028+00:00","status":404,"error":"Not Found","message":"Forhold ikke funnet.","path":"/api/samhandler/tjenestepensjon/forhold/2956"}</faultstring>`

Reply:
Dette er et eksempel på svar tjenesten gav tidligere. Den bør fortsette å svare tilsvarende

```
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
   <soap:Body>
      <soap:Fault>
         <faultcode>soap:Server</faultcode>
         <faultstring>TP-forholdet finnes ikke i registeret</faultstring>
         <detail>
            <ns2:slettTPForhold_faultTjenestepensjonForholdIkkeFunnet xmlns:ns2="http://nav.no/elsam/registreretpforhold/V0_1">
               <errorMessage>TP-forholdet finnes ikke i registeret</errorMessage>
               <errorSource>MODULE: nav-cons-elsam-tptilb-registreretpforhold / COMPONENT: authorizeAndOrchestrate / IF(OP): RegistrereTPForhold(hentTPForholdListe) / REF: SamhandlerPartner IF(OP): Samhandler(hentSamhandler)</errorSource>
               <dateTimeStamp>2024-10-03T11:14:41.377Z</dateTimeStamp>
            </ns2:slettTPForhold_faultTjenestepensjonForholdIkkeFunnet>
         </detail>
      </soap:Fault>
   </soap:Body>
</soap:Envelope>
```